e2e_testing/torchscript/basic.py

```diff
@@ -512,3 +512,21 @@ def forward(self, x, y):
 @register_test_case(module_factory=lambda: TensorToInt())
 def TensorToInt_basic(module, tu: TestUtils):
     module.forward(torch.randint(10,[]), tu.rand())
+    
+class LogSoftmaxIntModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        torch.manual_seed(0)
```

Comment:
I noticed that this test and some other tests in this file set the random generator seed to zero. Is there a reason we want these tests to always run on the same input?

Reply:
This is to make sure the golden test and the real test would have the same parameters.

Reply:
actually... softmax doesn't have parameters so this might not be needed.

Reply:
I see, I have removed the random seed param.